es-design-system/pages/molecules/es-form-msg.vue

```diff
@@ -8,17 +8,35 @@
                 bootstrap-vue alert
             </b-link>
         </p>
-        <es-button @click="fakeFormMsg()">
-            Show Success
+
+        <es-button
+            @click="showSuccess = true">Show Success
+        </es-button>
+        <es-button
+            @click="showError = true">Show Error
         </es-button>
-        <es-button @click="fakeFormMsg(false)">
-            Show Error
+        <es-button
+            @click="showInfo = true">Show Info
         </es-button>
+
+        <es-form-msg
+            class="my-450"
+            :show="showSuccess"
+            :variant="'success'"
+            @hidden="showSuccess = false">Saved Successfully
+        </es-form-msg>
         <es-form-msg
             class="my-450"
-            :variant="formMsgVariant"
-            :message="formMsg"
-            @hidden="formMsg = ''" />
+            :show="showError"
+            :variant="'danger'"
+            @hidden="showError = false">The server responded with an error and we were unable to complete your request. Please try again
+        </es-form-msg>
+        <es-form-msg
+            class="my-450"
+            :show="showInfo"
+            :variant="'primary'"
+            @hidden="showInfo = false">Need Expert Advice? &nbsp;<a href="https://www.energysage.com/"> Schedule a free call with our Energy Advisor Team!</a>
```

Comment:
Nit: can we change this to "Need expert advice?" so messaging copy is consistently in sentence case?

"Energy Advisor" is a brand term so that should still be capitalized, but "team" should be lowercase, as well.